src/vite.rs

```diff
@@ -125,6 +125,7 @@ impl Development {
             };
             html! {
                 html lang=(self.lang) {
+                    (DOCTYPE)
```

Comment:
a few thoughts:

1. i think this should go outside of the `<html>` tag, i.e. just above it (it needs to be the first element, not within the first element)
2. should this also go into the production vite config?